backend/qemu.pm

```diff
@@ -595,6 +598,7 @@ sub start_qemu() {
         };
         if ($@) {
             print "$@\n";
+            print "WARNING: Can't switch NICVLAN number, independent tests may be running on the same network.\n\n";
```

Comment:
Is it safe to continue with warning? I thought dying here was ok, just error message was misleading.